core/adapters/copy_test.go

```diff
@@ -80,9 +80,7 @@ func TestCopy_Perform(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		test := tt
```

Comment:
Why is it gone from here?

Reply:
To make things simpler, we are going to use `t.Parallel()` for top level tests, but not for sub-runs of table driven tests. There can be some confusing behavior when you do that(see the "wat" above), and so we want to make a pattern to avoid any instances of nested parallelism in tests moving forward.